src/helpers/columnWidth/columnWidth.js

```diff
@@ -8,12 +8,22 @@ const { theme } = themeProvider;
 export const percentage = (props, breakpoint) =>
   (Math.abs(props[breakpoint]) / theme(props).columns) * 100;
 
+export const isHidden = (props, breakpoint) =>
+  (percentage(props, breakpoint) === 0 || props[breakpoint] === 'hidden');
```

Comment:
Probably can just do:

(props[breakpoint] === 0 || props[breakpoint] === 'hidden');

rather than calling the other percentage function?